docs/kb/kb0011-configuration-migration.md

```diff
@@ -0,0 +1,152 @@
+    Please reach out to CluedIn support if you would like this toolkit.
+- Service or Admin account to do the export and import
+- Access to the frontend whether it's public facing or private facing
+
+### Guide
+The product toolkit supports a number of methods of transferring configuration from one environment to another. The main two methods are running locally on your desktop, or alternatively using a pipeline platform such as Azure DevOps
+
+Below will guide you through both local and Azure DevOps, but you can technically use other methods.
+
+#### Locally
+
+This guide will be using the `import` and `export` scripts only. The toolkit also supports running the functions on their own, but this will not be covered as part of this documentation.
+
+To successfully run this, you will need the following information before proceeding:
+- Full url of your frontend
+- A backup path locally to store the json files exported
+- Id's of the data you want to backup
+
+{:.important}
+To get the Ids of specific parameters, you will need to use the frontend and navigate to the desired resource. In the url, you will then notice the id of a given object. This may be a guid or integer depending on what is viewed.
+
+##### Exporting configuration (Source system)
+
+1. Open up a `PowerShell`.
+1. In the `PowerShell` window, run the following: 
+
+    **Note**: We will be using the ficticious front end `https://cluedin-source.customer.com` in the example below.
+
+    ```powershell
+    $BaseURL = 'customer.com'
+    $Organization = 'cluedin-source'
+    $BackupPath = '/path/to/export-20240403'
+
+    /path/to/CluedIn.Product.Toolkit/Scripts/Export-CluedInConfig.ps1 \
+        -BaseURL ${BaseURL} \
+        -Organization ${Organization} \
+        -BackupPath ${$BackupPath}
+        # The below are all optional. Please see what values are accepted below. If running without any of the below.
+        # It will not export anything as everything defaults to export nothing.
+        # 
+        # -BackupAdminSettings      [switch] true | false (Default)
+        # -SelectVocabularies       [string] csv guids | None (Default)
+        # -SelectDataSets           [string] csv ints | All | None (Default)
+        # -SelectRules              [string] csv guids | All | None (Default)
+        # -SelectExportTargets      [string] csv guids | All | None (Default) 
+        # -SelectStreams            [string] csv guids | All | None (Default)
+        # -SelectGlossaries         [string] csv guids | All | None (Default)
+        # -SelectCleanProjects      [string] csv guids | All | None (Default)
+    ```
+
+    **Note**: `/path/to/` will differ on your local system. Please update this to be the local respective path.
+
+1. The script will launch and prepare to connect. You will be prompted for a username and password which will obtain a JWToken if successful. Once connect, it will attempt to export all the desired configuration specified above.
```

Comment:
```suggestion
1. The script will launch and prepare to connect. You will be prompted for a username and password which will obtain a JWToken if successful. Once connected, it will attempt to export all the desired configuration specified above.
```